alerta-covid/app/assets/stylesheets/areas.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Areas controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
you can remove this comments, if are not necessary

Reply:
comments were removed from this file